packages/react-components/components/Button.v2.tsx

```diff
@@ -0,0 +1,133 @@
+import Touchable from '@celo/react-components/components/Touchable'
+import { colors } from '@celo/react-components/styles/colors.v2'
+import { fontStyles } from '@celo/react-components/styles/fonts.v2'
+import { debounce } from 'lodash'
+import React, { useCallback } from 'react'
+import { StyleProp, StyleSheet, Text, View, ViewStyle } from 'react-native'
+
+const BUTTON_TAP_DEBOUNCE_TIME = 300 // milliseconds
+const DEBOUNCE_OPTIONS = {
+  leading: true,
+  trailing: false,
+}
+
+export enum BtnTypes {
+  PRIMARY = 'Primary',
+  SECONDARY = 'Secondary',
+  TERTIARY = 'Tertiary',
+}
+
+export enum BtnSizes {
+  Small = 'small',
```

Comment:
Nit, casing inconsistent with enum above. I recommend caps, tho I know we haven't been consistent about it in the app

Reply:
cool